gnovm/cmd/gno/lint.go

```diff
@@ -210,9 +252,11 @@ func catchRuntimeError(pkgPath string, addIssue func(issue lintIssue), action fu
 type lintCode int
 
 const (
-	lintUnknown  lintCode = 0
-	lintNoGnoMod lintCode = iota
+	lintUnknown lintCode = 0
+	lintGnoMod  lintCode = iota
```

Comment:
```suggestion
	lintUnknown lintCode = iota
	lintGnoMod
```

isn't more idiomatic that way?

Reply:
Fair point